.github/workflows/ci.yml

```diff
@@ -1,31 +1,33 @@
-name: "Continuous Integration"
-
-on:
-  pull_request:
+---
+name: Continuous Integration
+"on":
```

Comment:
Nit: if you know which version of YAML is being used, quotes can possibly be removed here. Starting from YAML 1.2 (released in 2009) `on` is no longer interpreted as a boolean value.

Reply:
Nice! Thanks for pointing that out, I did not know that.

I'm not sure which exact version we need to target, do I'd like to keep the linter as strict as possible for now.